crates/ruff_python_parser/src/snapshots/ruff_python_parser__lexer__tests__empty_fstrings.snap

```diff
@@ -22,7 +22,7 @@ expression: lex_source(source)
     (
         String {
             value: "",
-            kind: StringKind {
+            flags: StringKind {
```

Comment:
@AlexWaygood Was this suppose to be "AnyStringKind"? If so, I'll rename it to "AnyStringFlags" otherwise I'll rename it to "StringFlags".

Reply:
I've renamed it to "AnyStringFlags".

Reply:
Thanks!